packages/esm-commons-lib/src/workspace/ohri-workspace-utils.ts

```diff
@@ -22,7 +22,7 @@ export const launchOHRIWorkSpace = (props: WorkspaceContextProps) => {
   const workspaceName = props.workspaceName || 'ohri-forms-' + counter++;
 
   const close = () => {
-    return closeWorkspace(workspaceName);
+    return closeWorkspace(workspaceName, true);
```

Comment:
This second parameter was added to prompt a dialogue box before closing a workspace (for our case a form) if at all the form has new data